src/test/resources/esapi/ESAPI-CommaValidatorFileChecker.properties

```diff
@@ -1,5 +1,35 @@
-#
 # OWASP Enterprise Security API (ESAPI) Properties file -- TEST Version
+#############################################################################
+#
+#   WARNING     WARNING     WARNING     WARNING     WARNING     WARNING
+#
+#   #######                                 #     #
+#      #     ######   ####    #####         #     #  ######  #####    ####
+#      #     #       #          #           #     #  #       #    #  #
+#      #     #####    ####      #           #     #  #####   #    #   ####
+#      #     #            #     #            #   #   #       #####        #
+#      #     #       #    #     #             # #    #       #   #   #    #
+#      #     ######   ####      #              #     ######  #    #   ####
+#   
+#   This is NOT the version of ESAPI.properties that you are looking for.
+#   Do NOT use this for your production applications!!!!!
+#
+#   That is over in the 'configuration/esapi/ESAPI.properties' file. You
+#   should retrieve THAT version from the official GitHub report from
+#       https://github.com/ESAPI/esapi-java-legacy/blob/develop/configuration/esapi/ESAPI.properties
+#   but make sure that you select it from the 'master' branch (which will
+#   correspond to the latest official ESAPI relase). Sorry for the
+#   inconvenience. We are trying to figure out how to get it to the official
+#   "esapi-<releaseVersion>-sources.jar fiel available from Maven Central, but
+#   in the meantime, you will have to get it from GitHub.
+#
+#   PLEASE do not base your production use of ESAPI on this TEST version of
+#   ESAPI.properties as this test version has been dummed down in several places
```

Comment:
Dummed?  was that on purpose!  :-D  If it was it's genius!

Reply:
While I'd love to claim credit for the incredible irony, I'll have to blame it on too many 'Guiness'. ;-) Speaking of which, I can't believe I misspelled that and never noticed. (Okay, I think I maybe had fixed it in the private BitBucket repo.)